### ISSUE TITLE
Set focus of other filed on creation

### ISSUE THREAD

**Issue by SamKry** [SamKry/TillTotal#2]
When pressing the (+) button, the keyboard focus should be set to the new entity.

## Use .focused(_:equals:)
- [Apple Doc](https://developer.apple.com/documentation/musickit/artworkimage/focused(_:))
- [Example](https://www.hackingwithswift.com/quick-start/swiftui/how-to-make-a-textfield-or-texteditor-have-default-focus)
